BootloaderCorePkg/Library/FspSupportLib/FspSupportLib.c

```diff
@@ -33,6 +33,54 @@ GetFspNvsDataBuffer (
   return GetGuidHobData (HobListPtr, Length, &gFspNonVolatileStorageHobGuid);
 }
 
+/**
+  Find an FSP resource descriptor large enough to accomodate
+  requested length.
+
+  @param  HobListPtr   A HOB list pointer.
+  @param  Length       A pointer to the requested length. If suitable
+                       descriptor is found, length will be updated with
+                       resource length.
+  @retval              Reserved region start address.  0 if this region does not exist.
+ */
+UINT64
+GetFspAvailableSystemMem(
```

Comment:
This function tries to find available memory of given length?

Then should it be renamed - as FindLoaderMemory or something like that without the FSP word in it?

Reply:
It's looking in the FSP-M HOB list resource descriptors for available memory.